pkg/scheduler/objects/application_test.go

```diff
@@ -90,15 +90,15 @@ func TestNewApplication(t *testing.T) {
 	if app.rmEventHandler == nil {
 		t.Fatal("non nil handler was not set in the new app")
 	}
-	assertUserGroupResource(t, getTestUserGroup(), nil)
+	assertUserGroupResource(t, getTestUserGroup("testuser", []string{"testgroup"}), nil, nil, 0)
```

Comment:
Same as before: if you need the ability to check user and group separately you add a new function for it instead of changing 50+ callers.